modules/standard/Random.chpl

```diff
@@ -674,46 +661,56 @@ module Random {
+    .. code-block:: chapel
 
-    Several high-level methods are provided to generate random numbers or
+      var A: [1..n] int;
+      var rs = new randomStream(int);
+
+      forall (r, a) in zip(rs.next(A.domain), A) do a = r;
+
+    * Create a random stream for each task:
+
+    .. code-block:: chapel
+
+      var A: [1..n] int;
+
+      forall a in A with (var rs = new randomStream(int)) do a = rs.next();
+
+
+    The ``randomStream`` provides several methods to generate random numbers or
     to manipulate arrays using random numbers:
 
-        * :proc:`randomStream.fill` to fill an array with random numbers
-        * :proc:`randomStream.shuffle` to randomly re-arrange the elements of an
-          array
-        * :proc:`randomStream.permute` to create a random permutation of
-          an arrays domain, and store it in the array
-        * :proc:`randomStream.choice` to randomly sample from an array or
-          range (*unstable*)
-
-    Note that these methods have top-level counterparts that will internally
-    create a ``randomStream`` and then call the corresponding method on it —
-    convenient for one-off uses. To generate many random numbers, it is
+        * :proc:`randomStream.fill`: fill an array with random numbers
+        * :proc:`randomStream.shuffle`: randomly re-arrange the elements of
+          an array
+        * :proc:`randomStream.permute`: create a random permutation of an
+          array, domain, or range
+        * :proc:`randomStream.choose` randomly sample an element from an array,
+          domain or range
+        * :proc:`randomStream.sample` randomly sample elements from an array,
+          domain, or range (with or without replacement)
+
+    Note that the module provides top-level counterparts to these methods that
+    internally create a ``randomStream`` and then call the corresponding method
+    on it — convenient for one-off uses. To generate many random numbers, it is
     generally more efficient to create a ``randomStream`` and call the relevant
     method on it repeatedly.
 
-    An individual random number can be requested using :proc:`randomStream.getNext`
-    which will advance the stream to the next position and return the value at
-    that position. The position of the stream can also be manipulated using:
-
-      * :proc:`randomStream.skipToNth` to skip to a particular position in the
-        stream (*unstable*)
-      * :proc:`randomStream.getNth` to skip to a particular position in the
-        stream and return the value at that position (*unstable*)
-
-    A ``randomStream`` can be initialized with a user-provided seed value. Two
-    ``randomStream`` instances initialized with the same seed will produce
-    identical sequences of random numbers. When not provided explicitly, a seed
-    value will be generated in an implementation specific manner designed to
-    minimize the chance that two distinct instances of the ``randomStream``
-    will have the same seed.
```

Comment:
Does this paragraph about a user-provided seed value need to be preserved at all? Is this behavior no longer relevant?

Reply:
The initializer docs and the module-level docs have the same information, so I figured this was safe to remove. lmk if you disagree.